### PR TITLE
Moving some of the CR status to the playbook

### PR DESCRIPTION
**Pull request by jtaleric**
Now all workloads will have a "Starting" status. We need to implement
the Running and Complete for each.

This additionally adds Load,<workload>, Complete for YCSB.

### CODE REVIEW

File: roles/load-ycsb/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 ---
+
+- name: Get current state
```
Comment (aakarshg) [inline roles/load-ycsb/tasks/main.yml]: Don't think its needed here.

Comment (jtaleric) [reply roles/load-ycsb/tasks/main.yml]: yup, you are right.

File: roles/byowl/tasks/main.yml [hunk 2]
```diff
@@ -6,3 +6,23 @@
     state: present
     definition: "{{ lookup('template', 'workload.yml') | from_yaml }}"
   when: byowl.kind is not defined
+
+- name: Wait for fiod job to succeed...
```
Comment (dustinblack) [inline roles/byowl/tasks/main.yml]: s/fiod/byowl/

File: roles/byowl/tasks/main.yml [hunk 3]
```diff
@@ -6,3 +6,23 @@
...
+    kind: Job
+    api_version: batch/v1
+    name: '{{ meta.name }}-byowl'
+    namespace: "{{ operator_namespace }}"
+  register: byowl_jobs
+  until: "byowl_jobs | json_query('resources[].status.succeeded')"
+  retries: 1000
+  delay: 20
+
+- k8s_status:
```
Comment (dustinblack) [inline roles/byowl/tasks/main.yml]: Should this have a `when` condition based on the success of the 'wait for job to succeed' task?

File: roles/fio-distributed/tasks/main.yml [hunk 4]
```diff
@@ -78,3 +78,23 @@
...
+    kind: Job
+    api_version: batch/v1
+    name: '{{ meta.name }}-fio-client'
+    namespace: "{{ operator_namespace }}"
+  register: fiod_jobs
+  until: "fiod_jobs | json_query('resources[].status.succeeded')"
+  retries: 1000
+  delay: 20
+
+- k8s_status:
```
Comment (dustinblack) [inline roles/fio-distributed/tasks/main.yml]: Same as above -- Do we want to set `Complete` to `true` even if the previous task times out?

Comment (jtaleric) [reply roles/fio-distributed/tasks/main.yml]: yes we do.

`Completed` does not necessarily mean `Success`.

File: roles/byowl/tasks/main.yml (hunk 3 above)
Comment (jtaleric) [reply roles/byowl/tasks/main.yml]: `Completed` does not necessarily mean `Success`.

**Pull request closed without merging**